install_wsl.ps1

```diff
@@ -0,0 +1,114 @@
+# WSL installation.
+#
+# Copyright (C) 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019 Gregory W. Chicares.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+#
+# http://savannah.nongnu.org/projects/lmi
+# email: <gchicares@sbcglobal.net>
+# snail: Chicares, 186 Belle Woods Drive, Glastonbury CT 06033, USA
+
+# If you have UnauthorizedAccess error, run this command in PowerShell:
+# Set-ExecutionPolicy Unrestricted
+#
+# Return back to the Restricted policy after installing:
+# Set-ExecutionPolicy Restricted
+#
+# More info: https:/go.microsoft.com/fwlink/?LinkID=135170
+
+
+# Enable Microsoft-Windows-Subsystem-Linux feature.
+if ((Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux).State -eq 'Disabled') {
+    'Enable Microsoft-Windows-Subsystem-Linux feature.'
+
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	# Close the current session because of the necessity of restarting.
```

Comment:
There is some inconsistent indentation here, please check for (unwanted) hard TABs.